cmd/flags.go

```diff
@@ -206,3 +209,16 @@ func EnvOrDefault(key string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func PrepareOSEnv() error {
```

Comment:
Why is this a separate step instead of simply being part of `NewLaunchEnv` (and `NewBuildEnv` depending on what decisions we make about the build environment). I am inferring that the idea is to ensure registry values are set in the environment before we read environment config (e.g `CNB_PLATFORM_API`), but isn't this already handled by the container runtime?